Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Select.hs

```diff
@@ -167,15 +167,15 @@ buildSearchReqLocation merchantId sessionToken address customerLanguage latLong@
               building = loc.building,
               area = loc.ward,
               full_address = decodeAddress loc
-            }
+            },False)
     _ -> getAddressByGetPlaceName merchantId sessionToken latLong
```

Comment:
```
_ -> do 
    mbAddress <- getAddressByGetPlaceName
    (address, isNearbyUsed)  <- maybe (pure . (, False)) ((, True) <$> getAddressByNearbySearchRequest) mbAddress
    pure (address, isNearbyUsed)```


and make getAddressByGetPlaceName as a `Maybe Address` function and make another function for addressByNearbySearchRequest.

Reply:
made  another function as`getAddressByNearBySearch` 
Since some field of Address is NULL only so couldn't use  this expression

do 
    mbAddress <- getAddressByGetPlaceName
    (address, isNearbyUsed)  <- maybe (pure . (, False)) ((, True) <$> getAddressByNearbySearchRequest) mbAddress
    pure (address, isNearbyUsed)